lib/main_common.pm

```diff
@@ -355,7 +355,8 @@ sub default_desktop {
 }
 
 sub load_shutdown_tests {
-    loadtest("shutdown/cleanup_before_shutdown");
+    # Schedule cleanup before shutdown only in cases the HDD will be published
+    loadtest("shutdown/cleanup_before_shutdown") if get_var('PUBLISH_HDD');
```

Comment:
Uh, sorry. I don't think this will actually work. I don't think there is a variable PUBLISH_HDD, rather PUBLISH_HDD_1 and alike :)

Reply:
You're right, fixing